features/admin.applications.v1/api/applications-settings.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2023-2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AsgardeoSPAClient, HttpClientInstance } from "@asgardeo/auth-react";
+import { HttpMethods } from "@wso2is/core/models";
+import { AxiosError, AxiosRequestConfig, AxiosResponse } from "axios";
+import { store } from "../../admin.core.v1/store";
+import { ApplicationsSettingsFormValuesInterface } from "../pages/applications-settings";
```

Comment:
Refer to https://github.com/wso2/identity-apps/pull/5964#discussion_r1566707642

Reply:
Fixed with [d54ec35](https://github.com/wso2/identity-apps/pull/5964/commits/d54ec35c7ee212f1039c9afea0052b20884b03c3)